Modules/_datetimemodule.c

```diff
@@ -3868,6 +3868,20 @@ date_weekday(PyObject *self, PyObject *Py_UNUSED(dummy))
     return PyLong_FromLong(dow);
 }
 
+static PyObject *
+date_timestamp(PyObject *self, PyObject *Py_UNUSED(dummy))
+{
+    PyObject *ordinal_obj = date_toordinal(self, NULL);
+    if (!ordinal_obj) {
+        return NULL;
+    }
+    long days_since_epoch = PyLong_AsLong(ordinal_obj) - 719163;
+    Py_DECREF(ordinal_obj);
+    double timestamp = days_since_epoch * 86400.0;
+
+    return PyFloat_FromDouble(timestamp);
```

Comment:
```suggestion
    long days_since_epoch = PyLong_AsLong(ordinal_obj) - 719163;
    Py_DECREF(ordinal_obj);
    double timestamp = days_since_epoch * 86400.0;
    return PyFloat_FromDouble(timestamp);
```

I think we can remove the blank line, or put a blankline before timestamp instead after. Up to you.